rfcs/xxxx-rfc-process/text.md

```diff
@@ -0,0 +1,72 @@
+
+In order to alleviate these questions and concerns, we have moved to Github to keep track of our RFCs moving forward.
+
+
+## When do you need to create an RFC?
+
+- Your project will take more than 1 month to complete.
+- Your project affects more than 1 discipline (i.e. requires frontend work and backend work).
+- Your idea outlines a core abstraction of the project upon which other features will be built.
+- Your project would benefit from an in-depth review and discussion from your peers.
+- You'd like to have a record of an investigative project that you did that recommends future work direction. (See [IETF Informational RFCs](https://www.ietf.org/standards/process/informational-vs-experimental/))
+
+Generally, err on the side of writing a small RFC over implementing a project without a written design.
+
+## How to create a new RFC
+
+The following are the steps to writing and approving a new RFC (not modifying):
+
+1. Create a new branch of the `cellxgene-rfcs` repo.
+2. Create a new directory under the `rfcs` directory named `XXXX-name-of-rfc`, where `name-of-rfc` is the name of your RFC, not those words directly.
+3. Copy over the contents of the `templates` directory into your newly created directory. This contains a folder for images and the RFC template.
+4. Rename `rfc_template.md` to `text.md` and edit it. Add images that you'd like to include in the `imgs` folder.
+5. When you are ready for review, create a pull request. Add your approvers (and any other people you'd like to review your work) as reviewers on the PR. Please note that [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com) and [Sara Gerber](mailto:sara.gerber@chanzuckerberg.com) are required Reviewers for all RFCs so that CZI Central Infra and CZI Security teams are aware of cellxgene technical work. They may be removed as reviewers if the RFC is non-technical. 
+6. As a courtesy, post a heads up on the [#org-sci-eng-single-cell](https://chanzuckerbergteam.slack.com/archives/GQGPP7925) Slack channel that there is a design available for review, specifically calling out the approvers. Generally, two weeks is nice for substantial designs to get reviewed, but use your best judgement on how much time you'd like for your design to be under review.
+7. Once all comments are addressed and the *approvers* have approved your pull request, **change the name of your directory** from `XXXX-name-of-rfc` to pick the next number (for example, this one will be `0000-rfc-process`).
+8. Merge your PR! Delete your branch! Close your design ticket! Code away!
+
+
+## How to update an existing RFC
+
+Minor and trivial changes may be made freely, with no announcement by the author(s) of the RFCs. Such changes include correcting typos, re-wordings, and the like.
+
+If the change is major, the following are the steps to revising an existing RFC:
+
+1. Create a new branch of the `cellxgene-rfcs` repo.
```

Comment:
We may benefit for a naming convention to differentiate between new RFC's and amendments to existing RFC's

Reply:
Great idea! I was trying to avoid being too prescriptive with the branch naming but it could be helpful! I'll add it in as a suggestion if that works for you?

Reply:
Added suggested branch names! Let me know what you think.